src/ipc/handler.c

```diff
@@ -1383,6 +1383,17 @@ void ipc_cmd(struct sof_ipc_cmd_hdr *hdr)
 	}
 }
 
+struct ipc_comp_dev *ipc_glb_get_comp_by_id(uint32_t comp_id)
+{
+	struct ipc_comp_dev *comp;
+
+	comp = ipc_get_comp_by_id(_ipc, comp_id);
+	if (!comp)
+		trace_ipc_error("ipc: comp %d not found", comp_id);
+
+	return comp;
+}
```

Comment:
what's the value of this wrapper ?

Reply:
This wrapper is not used anymore (deleted)